src/components/Steps.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import styled from 'styled-components';
+
+interface updateSelectFunc {
+  (key: number): void;
+}
+
+interface catalogsInfo {
+  name: string;
+  language: string;
+  maintainer: string;
+  topics: string[];
+}
+
+interface catalog {
+  name: string;
+  commit: string;
+}
+
+interface CatalogProps {
```

Comment:
Line 8, 15, and 20 should replace `catalog` with `step` as specified in documentation.